src/excmds.ts

```diff
@@ -4270,7 +4271,7 @@ const KILL_STACK: Element[] = []
 
 /** Hint a page.
 
-    @param option
+    @param args
```

Comment:
We should mention that most flags can be combined now, somewhere : )

Reply:
I reorganized the flags in the docs to mention this, as well as which flags are compatible/conflict with each other in 0a664c5d5cc10ae3e3b72f6f5726d1526bebf3c3